engines/standard_tasks/app/models/standard_tasks/reviewer_report_task.rb

```diff
@@ -9,5 +9,12 @@ def self.permitted_attributes
     has_one :paper_review, foreign_key: 'task_id'
 
     accepts_nested_attributes_for :paper_review
+
+    def send_emails
+      return unless previous_changes["completed"] == [false, true]
+      paper.editors.each do |editor|
+        ReviewerReportMailer.delay.notify_editor_email(task: self, recipient: editor)
```

Comment:
Please change this to pass the `id`s of these activerecord objects and not the objects themselves.

https://github.com/mperham/sidekiq/wiki/Best-Practices#1-make-your-jobs-input-small-and-simple